core/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -51,8 +48,20 @@ object RedundantBraces extends Rewrite {
 
   override def rewrite(code: Tree, ctx: RewriteCtx): Seq[Patch] = {
     import ctx.tokenTraverser._
+    import ctx.style.rewrite.{redundantBraces => settings}
     val builder = Seq.newBuilder[Patch]
     code.collect {
+      case t: Term.Interpolate if settings.stringInterpolation =>
+        t.parts.tail.zip(t.args).collect {
+          case (Lit(value), arg @ Term.Name(name)) =>
+            val openBrace = prevToken(arg.tokens.head)
+            val closeBrace = nextToken(arg.tokens.head)
+            (openBrace, closeBrace, value) match {
+              case (LeftBrace(), RightBrace(), "") =>
```

Comment:
Hi @olafurpg. I added all the test cases you mentioned. I'm just not sure that it does make big sense to test all the cases without braces! Since there is a check that args are inside braces! maybe one test here is enough?